doc/faq/readme.md

```diff
@@ -43,3 +43,111 @@ You will see this issue only if serial upload was not followed by a physical res
 
 Ref. [#1017](https://github.com/esp8266/Arduino/issues/1017), [#1107](https://github.com/esp8266/Arduino/issues/1107), [#1782](https://github.com/esp8266/Arduino/issues/1782)
 
+
+### I'm getting these Watchdog Resets "wdt reset"�How can I avoid them?
+The esp8266 is equipped with two watchdog timers: A software watchdog and a hardware watchdog. If you don't understand why microcontrollers need a watchdog (timer), you may first read [here](http://www.embedded.com/electronics-blogs/beginner-s-corner/4023849/Introduction-to-Watchdog-Timers).
+
+When your sketch is running�either in the `setup` or during the `loop`�you have to make sure to feed the watchdog periodically. Otherwise "it will bite" and the watchdog reset will stop your sketch. The basic operation is illustrated in the following picture:
```

Comment:
@pasko-zh, I think my comment was not to the point. Sorry, and let me be more specific. I would rephrase:
> Otherwise "it will bite" and the watchdog reset will stop your sketch. 

to:
Otherwise "it will bite" and the watchdog will reset your sketch. 
or:
Otherwise "it will bite" and the watchdog will restart your module. 

I think that "stop sketch" may confuse people, while "restart" or "reset" better explains what the watchdog does if not fed.

Thank you again for contributing to FAQ :+1:

Krzysztof